test/cql-pytest/test_ttl.py

```diff
@@ -37,18 +38,33 @@ def test_basic_default_ttl(cql, table_ttl_1):
         assert time.time() < start + 2
         time.sleep(0.1)
     # p1 should not have expired yet. By the way, its current ttl(v) would
-    # normally be exactly 999 now, but theoretically could be a bit lower in
+    # normally be exactly 99 now, but theoretically could be a bit lower in
```

Comment:
Isn't it `999`?
`p2` expires after 1 second, `p1` after 1000 seconds, so AFAIU `p1` should have 999 seconds left to live after 1 second

Reply:
Oops, I shouldn't have fixed it in this unrelated patch :-( I misread "100" in the code and didn't understand why the comment said 999 and not 99, but you're right, 999 was correct. I'll send a fixed version.